### PR TITLE
AMP-30054 fix javascript injection of scores from airtable

### PR DESCRIPTION
**Pull request by Sierangho**

### CODE REVIEW

File: flappy/leaderboard-prod.js [hunk 1]
```diff
@@ -1,47 +1,55 @@
...
+    .select({
+      fields: ["score", "vehicle"],
+      sort: [{ field: "score", direction: "desc" }],
+      maxRecords: 20,
+    })
+    .eachPage(
+      function page(records, fetchNextPage) {
         records.forEach(function (record) {
-            $('#hidden-list').append("<li>" + record.get('vehicle') + "<span id='score'>" + record.get('score') + "</span></li>");
+          var liNode = $("<li/>").text(record.get("vehicle"));
```
Comment (Sierangho) [inline flappy/leaderboard-prod.js:33]: actual non-formatting change - `record.get` calls inside "text"

File: flappy/leaderboard.js [hunk 2]
```diff
@@ -1,48 +1,58 @@
...
+      fields: ["score", "vehicle"],
+      sort: [{ field: "score", direction: "desc" }],
+      maxRecords: 100,
+    })
+    .eachPage(
+      function page(records, fetchNextPage) {
         records.forEach(function (record) {
-            $('#hidden-list').append("<li class='list-row'><div class='name'>" + record.get('vehicle') + "</div><div class='score'>" + record.get('score') +
-                "</div></li>");
+          var liNode = $("<li/>").attr("class", "list-row").append();
```
Comment (Sierangho) [inline flappy/leaderboard.js:33]: actual non-formatting change - `record.get` calls inside "text"

File: pacman/pacman-leaderboard.js [hunk 3]
```diff
@@ -10,20 +10,19 @@ function getHighscore() {
...
-        records.forEach(function(record) {
-          $("#highscore-list").append(
-            "<li>" +
-              record.get("name") +
-              "<span id='score'>" +
-              record.get("score") +
-              "</span></li>"
-          );
+        records.forEach(function (record) {
+          var liNode = $("<li/>").text(record.get("name"));
```
Comment (Sierangho) [inline pacman/pacman-leaderboard.js:20]: actual non-formatting change - `record.get` calls inside "text"

**Review by hacos — APPROVED** [review]
A bit rusty on my frontend code but looks good

**Pull request merged**